packages/contracts/test/transactionManager.spec.ts

```diff
@@ -486,8 +490,6 @@ describe("TransactionManager", function () {
       ).to.be.revertedWith("removeLiquidity: INSUFFICIENT_FUNDS");
     });
 
-    it.skip("should error if transfer fails", async () => {});
-
```

Comment:
We should return this test. We could use `smock` instead of writing a special ERC20 token transfer

Reply:
I already tested these cases in the lib test so we don't really smock it in transaction Manager